src/Task/Git/CommitMessage.php

```diff
@@ -110,39 +122,39 @@ public function run(ContextInterface $context): TaskResultInterface
             return TaskResult::createFailed(
                 $this,
                 $context,
-                'Commit message should not be empty.'
+                self::withCommitMessage('Commit message should not be empty.', $commitMessage),
```

Comment:
For every changed error message, you need to change the test-cases as well:
https://github.com/phpro/grumphp/blob/fb70da3dff23f55529ea2c13fd7c00a966278320/test/Unit/Task/Git/CommitMessageTest.php#L86-L94

Here you'll need to make sure the newly added 'original message' is applied to the error message so that it gets tested as well. (you could create a shortcut function for that as well inside that test class.

Reply:
I don't think a shortcut function is a good thing to test another shortcut function, so I just tested with some text on two validations. Tell me what you think about it.

Reply:
The tests pass because your test framework only test is the text is contained, I don't know if it's relevant to apply these modifications to all the tests (harder to read), but I'll do it if you think it's important.

Thank you for your review !